generic3g/vertical/FixedLevelsVerticalGrid.F90

```diff
@@ -79,5 +91,24 @@ logical function can_connect_to(this, src, rc)
        _UNUSED_DUMMY(src)
     end function can_connect_to
 
+    impure elemental logical function equal_FixedLevelsVerticalGrid(a, b) result(equal)
+       type(FixedLevelsVerticalGrid), intent(in) :: a, b
+
+       equal = a%standard_name == b%standard_name
+       if (.not. equal) return       
+       equal = a%units == b%units
+       if (.not. equal) return       
+       equal = size(a%levels) == size(b%levels)
+       if (.not. equal) return       
+       equal = all(a%levels == b%levels)
```

Comment:
Note to self (and others).  This pattern happens often enough throughout MAPL3 that I wonder if it is worth creating a special helper function.  The trick will be remembering what we called it.